packages/adapter-cloudflare/package.json

```diff
@@ -51,6 +51,6 @@
 	},
 	"peerDependencies": {
 		"@sveltejs/kit": "^2.0.0",
-		"wrangler": "^3.87.0 || ^4.0.0"
+		"wrangler": "^3.91.0 || ^4.0.0"
```

Comment:
since bumping the peer dep requires a major release, should we take this opportunity to drop support for wrangler 3?